### PR TITLE
Add scriptjs type defs

### PR DESCRIPTION
**Pull request by ssttevee**
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

for https://www.npmjs.com/package/scriptjs

### CODE REVIEW

**Comment by sandersn** [comment]
This is failing on the CI server.

**Comment by paulvanbrenk** [comment]
@ssttevee gentle ping. Can you fix the issues on the CI server? You're going to have to export a module 'scriptjs'
```
scriptjs/scriptjs-tests.ts(3,26): error TS2307: Cannot find module 'scriptjs'.
```

**Comment by ssttevee** [comment]
I'm not sure it works, neither import nor require seem to work...

The package is just 'scriptjs' on npm.

File: scriptjs/scriptjs.d.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Type definitions for scriptjs
+// Project: https://github.com/ded/script.js
+// Definitions by: Steve Lam <http://ssttevee.com>
```
Comment (paulvanbrenk) [inline scriptjs/scriptjs.d.ts]: Can you point this to your GitHub account?

**Comment by paulvanbrenk** [comment]
you need to export as follows:
```typescript
declare module 'scriptjs'{ 
    var $script: $script; 
    export = $script; 
}
```

**Pull request merged**